tests/e2e/performance/test_single_pvc_clone_perf.py

```diff
@@ -114,58 +122,70 @@ def test_clone_create_delete_performance(self, interface_type, pvc_size, file_si
             clone_yaml = constants.CSI_CEPHFS_PVC_CLONE_YAML
 
         # creating single clone ( or many one by one if max_mum_of_clones > 1)
-        logger.info(f"Start creating {max_num_of_clones} clones on {interface_type} PVC of size {pvc_size}GB.")
+        logger.info(f"Start creating {max_num_of_clones} clones on {interface_type} PVC of size {pvc_size} GB.")
 
         for i in range(max_num_of_clones):
             logger.info(f'Start creation of clone number {i + 1}.')
             cloned_pvc_obj = pvc.create_pvc_clone(sc_name, parent_pvc, clone_yaml, storage_size=pvc_size + "Gi")
             teardown_factory(cloned_pvc_obj)
             helpers.wait_for_resource_state(cloned_pvc_obj, constants.STATUS_BOUND, timeout)
+
             cloned_pvc_obj.reload()
             logger.info(f"Clone with name {cloned_pvc_obj.name} for pvc {parent_pvc} was created.")
             clones_list.append(cloned_pvc_obj)
             create_time = helpers.measure_pvc_creation_time(
                 interface_type, cloned_pvc_obj.name
             )
-            logger.info(f"Clone number {i+1} time creation is {create_time} secs.")
+            creation_speed = int(int(file_size) / create_time)
+            logger.info(f"Clone number {i+1} creation time is {create_time} secs.")
+            logger.info(f"Clone number {i+1} creation speed is {creation_speed} MB/sec.")
             clone_creation_time_measures.append(create_time)
 
-        logger.info(f"Finished creating {max_num_of_clones} clones on {interface_type} PVC of size {pvc_size}GB.")
+        logger.info(f"Finished creating and taking measures for {max_num_of_clones} "
+                    f"clones on {interface_type} PVC of size {pvc_size} GB.")
 
-        logger.info(f"Printing clone creation times for {max_num_of_clones} clones "
-                    f"on {interface_type} PVC of size {pvc_size}GB:")
+        logger.info(f"Printing clone creation time and speed for {max_num_of_clones} clones "
+                    f"on {interface_type} PVC of size {pvc_size} GB:")
 
         for i in range(max_num_of_clones):
-            logger.info(f"Clone number {i+1} creation time is {clone_creation_time_measures[i]} secs")
+            logger.info(f"Clone number {i+1} creation time is {clone_creation_time_measures[i]} secs.")
```

Comment:
@keesturam , You are right that in case of 1 clone it looks like redundant print . 
However, in case that max_num_of_clones is greated than 1, I'd like to have the measures printed twice: 
first - after each time the measures were successfully taken
and
second - in the end , to have all the measures together , in order to report them in a convenient way. 

My thought was that I do need the first print in case one of the multiple clone creations will fail. 
Then I do want to know what were the measures, ieven if the test does not reach the end. 

And the second print is needed because I do want all of the measures printed together. 

So what do you think about leaving it as is ?

Reply:
I think it will be better to move this to the end of the test and print this info together with the deletion info.

Reply:
@Avilir - will to it. Will print in the end of the test run at first all the creation times and then all the deletion times.